docs/user-guide/garbage-collector.md

```diff
@@ -0,0 +1,21 @@
+---
+---
+
+* TOC
+{:toc}
+
+## WARNING: Garbage Collector is an alpha feature, use it at your own risk!
+
+### What does the garbage collector do
+
+The garbage collector (GC) monitors the cluster and checks the metadata.ownerReferences field of each object. If all the owners present in metadata.ownerReferences do not exist in the cluster, the GC will request the API server to delete the object.
+
+When deleting an object, if you want to prevent the GC from deleting objects that depend on it, you can send the deletion request with `deleteOptions.orphanDependents=true`, then the GC will remove the object from its dependents' metadata.ownerReferences, so that the dependents will not be garbage collected.
```

Comment:
When deleting an object, you can prevent the GC from deleting that object's dependents by specifying `deleteOptions.orphanDependents=true` in the deletion request. This flag prevents garbage collection by removing the deleted object from its dependents' `metadata.ownerReferences` field.